src/test/java/com/trycloud/stepDefinitions/LogIn_stepDef_TD.java

```diff
@@ -0,0 +1,81 @@
+package com.trycloud.stepDefinitions;
+
+import com.trycloud.pages.*;
+import com.trycloud.utility.*;
+import io.cucumber.java.en.*;
+import org.apache.logging.log4j.*;
+import org.junit.*;
+import org.openqa.selenium.interactions.*;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class LogIn_stepDef_TD {
+    Logger LOG = LogManager.getLogger();
+    LoginPage loginPage=new LoginPage();
+    Actions action=new Actions(Driver.getDriver());
+    @Given("the user is on the login page")
+    public void the_user_is_on_the_login_page() {
+        LOG.info("user goes to login page");
```

Comment:
Empty method body

Reply:
You can confirm if it's login page, by checking the url or checking if there's login button present